gateways/c/fjage.c

```diff
@@ -257,6 +257,7 @@ static fjage_msg_t mqueue_get(fjage_gw_t gw, const char* clazz, const char* id)
   _fjage_gw_t* fgw = gw;
   fjage_msg_t msg = NULL;
   for (int i = fgw->mqueue_tail; i != fgw->mqueue_head && msg == NULL; i++) {
+    if (i >= QUEUE_LEN) i = i%QUEUE_LEN;
```

Comment:
Would it be better to just use a `i = (i+1)%QUEUE_LEN` as part of the for loop? It avoids a condition check and is more localized in terms of `i` update.

Reply:
Updated.